Engine/DataFeeds/UniverseSelection.cs

```diff
@@ -240,15 +241,28 @@ public SecurityChanges  ApplyUniverseSelection(Universe universe, DateTime dateT
                 Security security;
                 if (!pendingAdditions.TryGetValue(symbol, out security) && !_algorithm.Securities.TryGetValue(symbol, out security))
                 {
-                    security = universe.CreateSecurity(symbol, _algorithm, _marketHoursDatabase, _symbolPropertiesDatabase);
+                    security = SecurityManager.CreateSecurity(_algorithm.Portfolio, _algorithm.SubscriptionManager,
```

Comment:
no need to change -- I think this formatting showcases my earlier comment on preferring left-aligned code vs parenthesis aligned code. On the diff there's plenty of code scrolling off to the right, multiple parameters on each line, etc, just making it a little harder on the eyes

Reply:
also, a note for posterity, this is temporary code here as soon we'll replace all invocations of `SecurityManager.CreateSecurity` with an instance based approach, perhaps something like, `ISecurityService.CreateSecurity`

Reply:
I'll reformat it 👍